### PR TITLE
New Version Script

### PR DESCRIPTION
**Pull request by stevarino**
Put together a quick version updater script for adding a new version to minecraft-data.

My 1.19.21 PR (https://github.com/PrismarineJS/minecraft-data/pull/622) was put together with just this script. Not pretty, but it works. Happy to iterate on style if needed.

Currently untested for pc platform.

### CODE REVIEW

File: tools/js/incrementVersion.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+const fs = require('fs')
+const { join } = require('path')
+
+const root = join(__dirname, '..', '..')
+const data = join(root, 'data')
+const platforms = ['bedrock', 'pc']
```
Comment (extremeheat) [inline tools/js/incrementVersion.js:6]: Only bedrock uses the YAML files right now so this list should only contain bedrock

**Review by extremeheat — COMMENTED** [review]
Should be added to the updating documentation

File: tools/js/incrementVersion.js (hunk 1 above)
Comment (stevarino) [reply tools/js/incrementVersion.js:6]: Re: documentation - fully agree. But where should this be documented? minecraft-data, node-minecraft-data, or bedrock-protocol. To me I feel bedrock-protocol makes the most sense as that is where someone would likely look, but could either be standalone or a link to bedrock-specific instructions in minecraft-data. WDYT?

**Comment by stevarino** [comment]
Left a stub in case someone wants to add pc data later, as there are a lot of similar steps in pc version incrementing.

**Comment by extremeheat** [comment]
Lgtm, but have not tested yet.

**Comment by rom1504** [comment]
let's merge after it has been tried at least once

**Review by extremeheat — APPROVED** [review]
LGTM, tested script and seems to work

**Pull request merged**